modules/ding_gatewayf/modules/ding_gatewayf_registration/ding_gatewayf_registration.module

```diff
@@ -328,6 +334,19 @@ function ding_gatewayf_registration_acceptance_form() {
   return $form;
 }
 
+/**
+ * Sumbut handler that resets the information text.
+ */
+function ding_gatewayf_registration_information_reset($form, $form_state) {
+  $config = variable_get('ding_gatewayf_registration', FALSE);
+  if ($config) {
+    unset($config['information']);
+  }
+  variable_set('ding_gatewayf_registration', $config);
+
+  drupal_set_message(t('Information text have been reset to DDB default text'));
```

Comment:
This will also reset the text on the live site visible by end users until the editor has submitted the new text. Could we load the default text into the editor as a default value when clicking the button?

Reply:
I think the idea i very good. But i really don't known how to do this when its wysiwyg element with out writing a bounce of JS code?